capsules/src/usb/usbc_client_ctrl.rs

```diff
@@ -189,7 +161,7 @@ impl<'a, 'b, C: hil::usb::UsbController<'a>> ClientCtrl<'a, 'b, C> {
     }
 
     #[inline]
-    pub fn controller(&'a self) -> &'a C {
+    pub fn controller(&self) -> &'a U {
```

Comment:
I don't get how the lifetime parameter was removed in the input but not in the output.

Reply:
I don't see why the lifetime of the controller should be tied to the lifetime of the ClientCtrl struct? The controller reference is passed into the ClientCtrl struct when it is created.

I imagined the case where a controller with a static lifetime is passed to `ClientCtrl::new()` -- I don't think that should mean that a later reference to the ClientCtrl struct should have to be static in order for `controller()` to return a static reference to the controller.

Reply:
Makes sense. So the answer is that the previous lifetime was unnecessarily too restrictive, right?

Reply:
Yes, I believe so